syntax = "proto3";

package pegscanner.ping;

message PingResponse {
  string message = 1;
  string agent = 2;
  int64 timestamp_ms = 3;
}
